src/simulate.jl

```diff
@@ -18,8 +18,8 @@ function simulate(m::MixedModel, args...; kwargs...)
 end
 
 """
-    simulate!(rng::AbstractRNG, m::MixedModel{T}; β=m.β, σ=m.σ, θ=T[])
-    simulate!(m::MixedModel; β=m.β, σ=m.σ, θ=m.θ)
+    simulate!(rng::AbstractRNG, m::MixedModel{T}; β=fixef(m), σ=m.σ, θ=T[])
+    simulate!(m::MixedModel; β=fixef(m), σ=m.σ, θ=m.θ)
```

Comment:
I think it's worth documenting _somewhere_ the difference between the possible kinds of inputs to this function.  `coef`-style (all values in input model matrix order, pivoted out betas are `-0.0`) vs. `fixef`-style (only betas for full-rank columns, in pivot order)

Reply:
73d50ae0e96543728e41d48d3a6dc50d7362288e